packages/internal/test-util/src/lib/error-throwing-driver/error-throwing-driver.module.ts

```diff
@@ -39,7 +46,8 @@ export class ErrorThrowingDriverModule {
       providers: [
         {
           provide: errorThrowingDriverConfigToken,
-          useValue: allOptions,
+          useFactory: (logDriverConfig: LumberjackLogDriverConfig) => ({ ...logDriverConfig, ...allOptions }),
```

Comment:
suggestion: add explicit return type to avoid type errors.
```suggestion
          useFactory: (logDriverConfig: LumberjackLogDriverConfig): ErrorThrowingDriverConfig => ({ ...logDriverConfig, ...allOptions }),
```